cmd/humioctl/repos_block_ingest.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+func newReposBlockIngestCmd() *cobra.Command {
+	var allRepos bool
+
+	cmd := &cobra.Command{
+		Use:   "block-ingest [flags] [repository] <seconds>",
+		Short: "Block ingest for a repository",
+		Long: `Block ingest for one or all repositories for a specified number of seconds.
+
+Examples:
```

Comment:
There is an `Example` field in the `cobra.Command` struct where you could move these example invocations.